modules/ROOT/pages/manage-exchange-api-task.adoc

```diff
@@ -44,13 +45,46 @@ For example, if an API is defined with the `Implementation URI` set to `http://j
 |`/` | `https://domain.cloudhub.io/users` | `http://jsonplaceholder.typicode.com/users`
 | `/api` | `https://domain.cloudhub.io/api/users` | `http://jsonplaceholder.typicode.com/users`
 |===
-. Optionally, expand the *Advanced options* section and specify additional details:
-* Specify the validations and an API instance label for the API.
-* If you are creating an endpoint with proxy, also specify a scheme (HTTP or HTTPS) and a port, or accept the defaults.
-* Specify a response timeout, if required. 
+.. Optionally, expand the *Advanced options* section and specify additional details:
+* Optionally, In the *Proxy version* list, change the proxy version.
+* If you want to enable API specification testing for your API, check *Enable Console* and specify a different path in the *Console Path* field, for example, "/spec/*".
```

Comment:
Here we can also mention that this option will be available for Mule 4. 

Also, I am not sure if I would say _API specification testing_, maybe something like:
```
If you want to expose your API specification through the API Console, ...
```

Reply:
Done! Thanks! The sections are divided by Mule 3 and Mule 4 from the beginning. I don't think that we need to stress again.